test/verify/machineslib.py

```diff
@@ -1809,6 +1823,7 @@ def installWithErrorTest(dialog):
 
             # Disable selinux because it makes TFTP directory inaccesible and we don't want sophisticated configuration for tests
             self.machine.execute("if type selinuxenabled >/dev/null 2>&1 && selinuxenabled; then setenforce 0; fi")
+            self.addCleanup(self.machine.execute, "setenforce 1")
```

Comment:
Ugh, how ugly -- let's fix that properly with setting the correct type with `chcon`. But let's do that in a separate PR, I made a note in #13717